dataactvalidator/config/sqlrules/c23_award_financial_3.sql

```diff
@@ -15,20 +22,19 @@ SELECT
 		FROM award_financial_assistance as sub_afa
 		WHERE submission_id = {0} AND sub_afa.fain = af.fain and
 		COALESCE(sub_afa.assistance_type,'') in ('07','08')) AS original_loan_subsidy_cost_sum
-FROM award_financial AS af
+FROM award_financial_c23_3_{0} AS af
 JOIN award_financial_assistance AS afa
 		ON af.fain = afa.fain
 	  AND af.submission_id = afa.submission_id
-WHERE af.submission_id = {0}
 GROUP BY af.fain
 HAVING
 		(SELECT COALESCE(SUM(sub_af.transaction_obligated_amou::numeric),0) AS transaction_sum
-		FROM award_financial as sub_af WHERE submission_id = {0} AND sub_af.fain = af.fain) <>
+		FROM award_financial_c23_3_{0} as sub_af WHERE submission_id = {0} AND sub_af.fain = af.fain) <>
```

Comment:
The original was `FROM award_financial as sub_af WHERE submission_id = {0} AND sub_af.fain = af.fain)` so we want to pull from the temp table so the new should be `FROM award_financial_c23_3_{0} as sub_af WHERE  sub_af.fain = af.fain`